src/Build/Resources/Strings.resx

```diff
@@ -1973,4 +1976,10 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
       LOCALIZATION: {0} is a file path. {1} is a comma-separated list of target names
     </comment>
   </data>
+   <data name="DefaultSDKResolverError" xml:space="preserve">
+    <value>MSB4276: The default SDK resolver failed to resolve SDK "{0}" because directory "{1}" did not exist.</value>
+  </data>
+  <data name="SDKResolverReturnedNull" xml:space="preserve">
```

Comment:
Change to "SDK resolver "{0}" returned null." then make bug to replace with this again after updating nuget warnings --> errors.